### PR TITLE
Third update to intro.md

### PR DESCRIPTION
**Pull request by luzeal**
I have created another pull request to revise the introduction page. It replaces : #871, which will eventually need to be dropped. 
This pull request is linked to issue [5357](
https://github.com/QubesOS/qubes-issues/issues/5357).

Besides some formatting changes (one line per sentence ; ### Title ###), I have slightly modified the text in the "why" section. 

Thanks !

### CODE REVIEW

**Comment by luzeal** [comment]
> Why is there so much HTML? Please write in Markdown, as previously requested.

I treated the intro page the same way as the Downloads page or the main page, which are relying heavily on HTML formatting. This allows for a more visually appealing layout (as seen here https://github.com/QubesOS/qubes-issues/issues/5357#issuecomment-558716800). Although it would be possible to roll back to plain markdown, it would obviously break the current layout, which I consider to be a plus. See also @marmarek's remark on that question : https://github.com/QubesOS/qubes-doc/pull/871#issuecomment-559338444.

**Comment by andrewdavidwong** [comment]
> > Why is there so much HTML? Please write in Markdown, as previously requested.
> 
> I treated the intro page the same way as the Downloads page or the main page, which are relying heavily on HTML formatting. This allows for a more visually appealing layout (as seen here [QubesOS/qubes-issues#5357 (comment)](https://github.com/QubesOS/qubes-issues/issues/5357#issuecomment-558716800)). Although it would be possible to roll back to plain markdown, it would obviously break the current layout, which I consider to be a plus. See also @marmarek's remark on that question : [#871 (comment)](https://github.com/QubesOS/qubes-doc/pull/871#issuecomment-559338444).

Ok, that's fine. It's worth noting that the principled way to do this would be to have a separate layout file, following the standard separation between CSS, HTML, and Markdown. The Downloads page is indeed an exception to this, but it also doesn't have much prose content.

**Comment by andrewdavidwong** [comment]
Please fix the spelling errors reported by Travis CI:

https://travis-ci.com/QubesOS/qubes-doc/builds/138767678#L4672

```
The following new unknown spellings were introduced:
    +Suprinsingly
    +Templating
    +Utilise
    +activites
    +attachements
    +intertwinded
    +miriad
In addition, the following suspicious spellings were eliminated:
    -Aren
    -Malware
    -antivirus
    -unsecure
The introduced suspects in context are:
./_doc/introduction/intro.md:116:        <p>Suprinsingly, personal computing devices aren't designed to offer means to enforce the same kind of isolation that people enjoy in the physical world.</p>
./_doc/introduction/intro.md:46:      <h3>Templating system</h3>
./_doc/introduction/intro.md:75:      <p>Utilise <a href="/doc/split-gpg/">Split GPG</a> to store private GPG keys in an AppVM</p>
./_doc/introduction/intro.md:100:        <p>Throughout their lives, individuals engage in various activites such as going to school, working, voting, taking care of their families or visiting with friends. </p> 
./_doc/introduction/intro.md:101:        <p>These activites are spatially and temporally bound : they happen in isolation of one another, in their own compartments, which often represent an essential safeguard, such as in the case of voting.</p> 
./_doc/introduction/intro.md:128:        <p>Computing should remain a activity where mistakes can be made and where users can explore the web freely, downloading attachements and clicking on links without having to constantly evaluate a miriad of risk factors.</p> 
./_doc/introduction/intro.md:102:        <p>In one's digital life, the situation is quite different : each activity, often intertwinded with its real-life counterpart, tends to happen on a single computing device.</p>
./_doc/introduction/intro.md:128:        <p>Computing should remain a activity where mistakes can be made and where users can explore the web freely, downloading attachements and clicking on links without having to constantly evaluate a miriad of risk factors.</p> 
```

**Comment by luzeal** [comment]
> > > Why is there so much HTML? Please write in Markdown, as previously requested.
> > 
> > 
> > I treated the intro page the same way as the Downloads page or the main page, which are relying heavily on HTML formatting. This allows for a more visually appealing layout (as seen here [QubesOS/qubes-issues#5357 (comment)](https://github.com/QubesOS/qubes-issues/issues/5357#issuecomment-558716800)). Although it would be possible to roll back to plain markdown, it would obviously break the current layout, which I consider to be a plus. See also @marmarek's remark on that question : [#871 (comment)](https://github.com/QubesOS/qubes-doc/pull/871#issuecomment-559338444).
> 
> Ok, that's fine. It's worth noting that the principled way to do this would be to have a separate layout file, following the standard separation between CSS, HTML, and Markdown. The Downloads page is indeed an exception to this, but it also doesn't have much prose content.

Thanks for your understanding Andrew, and the piece of advice. With regard to the other message below : I will correct the spellings errors during the course of this week. I hope there is no rush. Best

File: introduction/intro.md [hunk 1]
```diff
@@ -13,166 +13,124 @@ redirect_from:
...
-but there are still risks to consider. Briefly, here are some of the main
-pros and cons of this approach relative to Qubes:
-
-<div class="focus">
-  <i class="fa fa-check"></i> <strong>Pros</strong>
+  <div class="row">
+      <div class="col-lg-3 col-md-3 text-left">
+          <p>Qubes OS is a free and open-source security-oriented operating system meant for single-user desktop computing.</p> 
+          <p>Qubes OS leverages <a href="https://wiki.xen.org/wiki/Xen_Project_Software_Overview/">xen-based virtualization</a> to allow for the creation and management of isolated virtual machines called <a href="/doc/glossary#qube">qubes</a>. 
+          Qubes, which are also referred to as <a href="/doc/glossary#qube/">domains</a> or compartments, have specific :</p>
```
Comment (marmarek) [inline introduction/intro.md]: This link is broken, it should be `/doc/glosary/#qube` instead.

Comment (luzeal) [reply introduction/intro.md]: Sorry for the error and thank you for reviewing my pull request. I have corrected the broken link in patch-4.

**Pull request merged**